nss-day-cohort-48/giffygram-jumpstation
# Added sections to rendered giffygram. Also added create post button.

#### Changes Made

1. Modified file main.js and giffygram.js to include html rendering of website sections. Also added create post button functionality.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout nrh-main
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   Review main landing page after you login and test the button to make sure when it is clicked it re-renders with the input fields.
4. View code file.
   Confirm that code has no errors.
